tests/test-recipes/metadata/_noarch_python2/meta.yaml

```diff
@@ -0,0 +1,23 @@
+package:
+  name: noarch_python2_only_test_package
+  version: "1.0"
+
+source:
+  path: ./noarch_python_test_package
+
+build:
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+  noarch: python
+  entry_points:
+    - noarch_python_test_package_script = noarch_python_test_package:main
+
+requirements:
+  build:
+    - python >=2,<3
```

Comment:
This allows python 3 pre-release versions, and might be fragile.  Please prefer

``python >=2,<3.0a0``

Reply:
Should we have the `.0` or should it just be `3a0`?

Reply:
I think you need the .0, but I'm not certain.